src/WebInterface.hs

```diff
@@ -49,7 +66,7 @@ renderPage pageTitle bodyHtml = renderHtml $ docTypeHtml $ do
     meta ! name "viewport" ! content "width=device-width, initial-scale=1"
     meta ! name "robots" ! content "noindex, nofollow"
     title $ toHtml pageTitle
-    style $ toHtml stylesheet
+    link ! rel "stylesheet" ! href (toValue stylesheetUrl)
```

Comment:
If we have the base64-encoded sha256sum already anyway, we might as well add [an `integrity`](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) attribute here.

Reply:
Done! This was pretty easy to do. We do need the non-URLsafe version of that digest, but that was pretty easy to